File: src/main/java/no/novari/flyt/egrunnerverv/gateway/dispatch/repository/InstanceHeadersRepository.java
package no.novari.flyt.egrunnerverv.gateway.dispatch.repository;

import no.novari.flyt.egrunnerverv.gateway.dispatch.model.InstanceHeadersEntity;
import org.springframework.data.jpa.repository.JpaRepository;

public interface InstanceHeadersRepository extends JpaRepository<InstanceHeadersEntity, String> {
}

File: src/main/kotlin/no/novari/flyt/egrunnerverv/gateway/dispatch/repository/InstanceHeadersRepository.kt
package no.novari.flyt.egrunnerverv.gateway.dispatch.repository

import no.novari.flyt.egrunnerverv.gateway.dispatch.model.InstanceHeadersEntity
import org.springframework.data.jpa.repository.JpaRepository

interface InstanceHeadersRepository : JpaRepository<InstanceHeadersEntity, String>
